### PR TITLE
Update 20.04.0 release

### PR DESCRIPTION
**Pull request by philippotto**
- [X] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [X] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable
- [X] Ready for review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,36 @@ and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MIC
...
+[Commits](https://github.com/scalableminds/webknossos/compare/20.04.0...HEAD)
+
+ ### Added
+-
+
+ ### Changed
+-
+
+ ### Fixed
+- Users only get tasks of datasets that they can access. [#4488](https://github.com/scalableminds/webknossos/pull/4488)
```
Comment (fm3) [inline CHANGELOG.md:19]: is that really in the code base? I thought we had reverted that

Comment (philippotto) [reply CHANGELOG.md:19]: Oh man, this is really getting out of hand. I cherrypicked the revert-commit you mentioned, but git says "The previous cherry-pick is now empty, possibly due to conflict resolution.". I'll change it manually, but this whole process seems very error-prone to me. I'll think about how to enforce that nobody merges into the "released" changelog sections (which can easily happen due to git merge).

**Comment by philippotto** [comment]
Ok, @jstriebel and @fm3, if you agree, I'd tag the latest commit of this PR as 20.04.1. Afterwards, I'll pull the master in (and fix the changelog again). Then, everything should work :crossed_fingers:

**Comment by philippotto** [comment]
> LGTM  You could also release this as `20.04.0`, since it was never tagged before

Good point, I only created the tag locally and didn't push it :man_facepalming:

**Comment by philippotto** [comment]
Release is created. Will now resolve the merge conflicts to merge this PR again.

**Pull request merged**